GX-TimeFrequency/data/minimax_grids/10_freq_time_points.dat

```diff
@@ -0,0 +1,227 @@
+# Published in J. Chem. Theory Comput. 2021, 17, 3, 1662-1677	
+#  Minimax grids with 10 points, for different energy ranges (Erange)	
+# Grid-points: 10.  N Eranges: 17	
+	
+# Column1: Frequency    column2: weight	# Column3: Time    column4: weight
+	
+Erange 0.0000 10.0000	
+	
+ 0.1691355148        1.3668906607	0.0330697624        0.0851548885
+ 0.5284642103        1.5376910384	0.1768078438        0.2041221377
+ 0.9545511712        1.9085090684	0.4464853449        0.3387624704
+ 1.5047418958        2.5454826105	0.8637994095        0.5020218013
+ 2.2597115867        3.5752953953	1.4665304201        0.7136389791
+ 3.3454936705        5.2519519792	2.3172114001        1.0042283384
+ 4.9862966533        8.1659529947	3.5172089430        1.4225951253
+ 7.6650093063       14.0412676455	5.2328732181        2.0553775440
+12.7796542255       29.9635632405	7.7617287680        3.1013168392
+27.2864801899      113.9616484063	11.7952949791        5.3215604241
```

Comment:
Formatting is off here. It's not a bit deal ( I assume things still parse) but would be nice to clean up if you're able to do it in an automated way/without too much work?

Reply:
This applies to all files - if you can do efficiently